### PR TITLE
Integrate Gradle and create a simple test

### PR DESCRIPTION
**Pull request by soer7022**
I have integrated Gradle and added a sanity check.

NOTE: We're using JUnit4 and not JUnit5, this is mainly due to me being unable to get it to work 😂

### CODE REVIEW

File: src/test/kotlin/TestSanity.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package test.kotlin
+
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+
+
+class TestSanity {
+    @Test
+    fun twoPlusOneEqualsThree() {
+        assertEquals("2+1 was not equal 3",3,2+1)
```
Comment (anbclausen) [inline src/test/kotlin/TestSanity.kt:10]: We're using assertThat from org.hamcrest.MatcherAssert.assertThat in SWEA.

**Review by anbclausen — COMMENTED** [review]
We're using assertThat from org.hamcrest.MatcherAssert.assertThat in SWEA. Any thoughts on this? Should we do that or continue with standard JUnit?

**Comment by soer7022** [comment]
https://stackoverflow.com/questions/27256429/is-org-junit-assert-assertthat-better-than-org-hamcrest-matcherassert-assertthat looks like Hamcrest is preferred, I will fix it.

**Review by anbclausen — APPROVED** [review]
Beautiful.

**Review by dangerstone — APPROVED** [review]

**Pull request merged**